examples/migrations_example/test/drift/default/migration_test.dart

```diff
@@ -0,0 +1,204 @@
+// ignore_for_file: unused_local_variable, unused_import
+// GENERATED CODE, DO NOT EDIT BY HAND.
+import 'package:drift/drift.dart';
+import 'package:drift_dev/api/migrations.dart';
+import 'package:migrations_example/database.dart';
+import 'package:test/test.dart';
+import 'schemas/schema.dart';
+
+import 'schemas/schema_v1.dart' as v1;
+import 'schemas/schema_v2.dart' as v2;
+import 'validation/v1_to_v2.dart' as v1_to_v2;
+import 'schemas/schema_v3.dart' as v3;
+import 'validation/v2_to_v3.dart' as v2_to_v3;
+import 'schemas/schema_v4.dart' as v4;
+import 'validation/v3_to_v4.dart' as v3_to_v4;
+import 'schemas/schema_v5.dart' as v5;
+import 'validation/v4_to_v5.dart' as v4_to_v5;
+import 'schemas/schema_v6.dart' as v6;
+import 'validation/v5_to_v6.dart' as v5_to_v6;
+import 'schemas/schema_v7.dart' as v7;
+import 'validation/v6_to_v7.dart' as v6_to_v7;
+import 'schemas/schema_v8.dart' as v8;
+import 'validation/v7_to_v8.dart' as v7_to_v8;
+import 'schemas/schema_v9.dart' as v9;
+import 'validation/v8_to_v9.dart' as v8_to_v9;
+import 'schemas/schema_v10.dart' as v10;
+import 'validation/v9_to_v10.dart' as v9_to_v10;
+import 'schemas/schema_v11.dart' as v11;
+import 'validation/v10_to_v11.dart' as v10_to_v11;
+
+void main() {
+  driftRuntimeOptions.dontWarnAboutMultipleDatabases = true;
+  late SchemaVerifier verifier;
+
+  setUpAll(() {
+    verifier = SchemaVerifier(GeneratedHelper());
+  });
+
+  test(
+  "default - migrate from v1 to v2",
+  () => testStepByStepigrations(
+    from: 1, to: 2, verifier: verifier, createOld: (e) => v1.DatabaseAtV1(e),
+    createNew: (e) => v2.DatabaseAtV2(e), openTestedDatabase: (e) => Database(e),
+    createItems: (b, oldDb) {
+      b.insertAll(oldDb.users, v1_to_v2.usersV1);
```

Comment:
Doesn't this have to be async?

Reply:
No
<img width="469" alt="image" src="https://github.com/user-attachments/assets/5b3633e5-a180-41a7-8366-c793b97bde20">